docs/authentication-contribute.md

```diff
@@ -0,0 +1,8 @@
+# Authentication
+
+## Adding a new authentication
+
+To add a new authentication method:
```

Comment:
I think the main steps are more complicated than this, right now we're really built around google. Maybe can we say for people to contact me and i'd be happy to help if there is interest?